api/graphql/schema.graphql

```diff
@@ -883,12 +883,12 @@ type Query {
     @find(model: "User")
     @guard
     @can(ability: "view", query: true, model: "User")
-    @deprecated(reason: "Removing in favour of user query")
+    @deprecated(reason: "applicant is deprecated. Use user instead.")
```

Comment:
The queries `applicant` and `applicants` weren't part of the acceptance criteria, but makes sense to use the same style of `reason` I guess. Do you mind updating the copy for the other reasons where the format "oldField is deprecated. Use newField instead." makes sense?

## Screenshot
![Screen Shot 2023-08-17 at 12 54 37](https://github.com/GCTC-NTGC/gc-digital-talent/assets/3046459/eee4d640-b6c0-491b-a1d0-31b17c11c413)

Reply:
Makes sense to me! Updated in [standardize deprecation reason messages](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7641/commits/7c58ca613b07ecf338374e8775fe8148991478ae)

Also, I found another item that was deprecated that can be removed (I think). Created a ticket https://github.com/GCTC-NTGC/gc-digital-talent/issues/7645